src/System Application/App/Extension Management/src/ExtensionManagement.Page.al

```diff
@@ -298,10 +354,26 @@ page 2500 "Extension Management"
                 actionref(Unpublish_Promoted; Unpublish) { }
                 actionref(SetupApp_Promoted; SetupApp) { }
                 actionref("Download Source_Promoted"; "Download Source") { }
-                actionref("Open Source in VS Code_Promoted"; "Open Source in VS Code") { }
                 actionref("Learn More_Promoted"; "Learn More") { }
                 actionref(Refresh_Promoted; Refresh) { }
             }
+
+            group("Develop in VS Code_Promoted")
+            {
+                Caption = 'Develop in VS Code';
+                ToolTip = 'Sets up a new of configures your existing local AL project in VS Code for extension development.';
```

Comment:
Is the tooltip needed? No other Promoted group has it. If you keep it, it should be synched with line 267.